data/tuyoData.json

```diff
@@ -4,6 +4,36 @@
             "quote": "Beside the coals of the dying fire, within the trampled borders of our abandoned camp, surrounded by the great forest of the winter country, I waited for a terrible death.",
             "chapter": "Chapter 1",
             "percentage": 1
+        },
+        {
+            "quote": "At last the warleader dismounted. He was far taller than I; even taller than most of his own people. He looked cruel to me, with a hard set to his mouth.",
+            "chapter": "Chapter 1",
+            "percentage": 1
+        },
+        {
+            "quote": "He said to that man, \"We must have pressed them even harder than we knew, if they've left a tuyo for us.\"",
+            "chapter": "Chapter 1",
+            "percentage": 1
+        },
+        {
+            "quote": "I must have jerked in outrage, for he turned quickly to look at me again.",
+            "chapter": "Chapter 1",
+            "percentage": 1
+        }
+        ,        {
+            "quote": "He tilted his head in surprise, perhaps at my words, or perhaps because I spoke darau at all.",
+            "chapter": "Chapter 1",
+            "percentage": 1
+        },
+        {
+            "quote": "_That_ drove everything else out of my mind. The shock was greater because I had just been thinking my brother and our people were safe. For a moment, I could not answer. Then I found my voice.",
+            "chapter": "Chapter 1",
+            "percentage": 1
+        },
+        {
+            "quote": "When my brother brought word of what had happened, my mother would not think of me sitting in an uncomfortable Lau chair in a twoo-warm Lau tent, eating too-spicy barley stew.",
```

Comment:
Typo (twoo)

Reply:
Fixed in [4e175cb](https://github.com/RebeccaAnne/QuoteBot/pull/22/commits/4e175cbaf6f975c85dbb0c23ed366f4f02d344cb)!